04-understanding-survey-data-documentation.Rmd

```diff
@@ -213,14 +202,14 @@ cps_state %>%
 targetpop <- cps_state %>%
   as_tibble() %>%
   filter(PRTAGE >= 18,
-         PRCITSHP %in% (1:4)) %>%
+         PRCITSHP %in% (1:4)) %>% 
```

Comment:
We should provide detail to the reader why we're doing this and what it represents.